settings.gradle

```
@@ -0,0 +1,3 @@
+rootProject.name = 'TheCamp'
+include 'app'
```

```suggestion
include ':app'
```
이 표현과는 어떤 차이가 있는 건가요?